src/main/java/com/example/bankservice/model/Account.java

```diff
@@ -0,0 +1,40 @@
+package com.example.bankservice.model;
+
+import com.example.bankservice.model.enums.Currency;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.Data;
+import lombok.ToString;
+
+@Data
+@Entity
+@Table(name = "accounts")
+public class Account {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(unique = true, name = "account_number")
+    private String accountNumber;
+    private double balance;
```

Comment:
just add constrain for min number

Reply:
@YuraBondarenko Let's use BigDecimal for money fields